app/code/Magento/ConfigurableProduct/Test/Mftf/Test/EndToEndB2CGuestUserTest.xml

```diff
@@ -9,6 +9,215 @@
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
     <test name="EndToEndB2CGuestUserTest">
+    <before>
```

Comment:
Please, revert changes, files in the ConfigurableProduct module are part of Magento core, not MSI project

Reply:
Reverted changes